libs/utils/utils.js

```diff
@@ -24,6 +24,23 @@ export function getMetadata(name) {
   return meta && meta.content;
 }
 
+export default function createTag(tag, attributes, html) {
+  const el = document.createElement(tag);
+  if (html) {
+    if (html instanceof HTMLElement) {
+      el.append(html);
+    } else {
+      el.insertAdjacentHTML('beforeend', html);
+    }
+  }
+  if (attributes) {
+    Object.keys(attributes).forEach((key) => {
+      el.setAttribute(key, attributes[key]);
```

Comment:
fyi: can use Object entries - no need to change tho
```suggestion
    Object.entries(attributes).forEach(([key, val]) => {
      el.setAttribute(key, val);
```